cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -982,9 +982,18 @@ instance ToObject (TraceTxSubmissionInbound (GenTxId blk) (GenTx blk)) where
   toObject _verb _ =
     mkObject [ "kind" .= String "TraceTxSubmissionInbound" ]
 
-instance ToObject (TraceTxSubmissionOutbound (GenTxId blk) (GenTx blk)) where
-  toObject _verb _ =
-    mkObject [ "kind" .= String "TraceTxSubmissionOutbound" ]
+instance (Show (GenTx blk), Show (GenTxId blk))
+      => ToObject (TraceTxSubmissionOutbound (GenTxId blk) (GenTx blk)) where
+  toObject _verb (TraceTxSubmissionOutboundRecvMsgRequestTxs txids) =
+    mkObject
+      [ "kind" .= String "TraceTxSubmissionOutboundRecvMsgRequestTxs"
+      , "txIds" .= String (pack $ show txids)
+      ]
+  toObject _verb (TraceTxSubmissionOutboundSendMsgReplyTxs txs) =
```

Comment:
if the list of transactions can be large potentially, then it would make sense to only output the transactions ids for "MaximumVerbosity". Similarly, could the tx ids be shortened for "NormalVerbosity"?
This is more a thought than a review.

Reply:
At the moment, I believe we'll only see a max of 2 transactions per one of these traces. However, I've made the changes such that the txids and txs will only be output for `MaximumVerbosity`.

I'll also open an issue to revisit the verbosity-specific output for this instance (shortened txids, etc.).